src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -931,6 +935,23 @@ def forward(
             cross_attentions=transformer_outputs.cross_attentions,
         )
 
+    @staticmethod
+    def _reorder_cache(past: Tuple[Tuple[torch.Tensor]], beam_idx: torch.Tensor) -> Tuple[Tuple[torch.Tensor]]:
+        """
+        This function is used to re-order the :obj:`past_key_values` or :obj:`mems` cache if
```

Comment:
Same here, it would be nice if we can clean up the docs

Reply:
I cleaned it as well.